bin/verilator

```diff
@@ -3405,6 +3405,42 @@ behavior.  See the test_regress/t/t_dpi_display.v file for an example.
 Same as C<sformat> in configuration files, see L</"CONFIGURATION FILES">
 for more information.
 
+=item /*verilator split_var*/
+
+Attached to a variable or a net declaration. It breaks the variable into
+multiple pieces. The following data types are supported.
+
+  Packed array and packed struct consist of Integer types (reg, logic, bit, byte, int...)
+  unpacked array of data type above
+
+Verilator will internally convert a variable with the metacomment such as:
+
+    logic [7:0] x [0:1];  /*verilator split_var*/
+
+to packed variables below.
+
+    logic [7:0] x__BRA__0__KET__; /*verilator split_var*/
+    logic [7:0] x__BRA__1__KET__; /*verilator split_var*/
+
+Note that the generated packed variables still have split_var metacomment because
+they will be split into further smaller pieces accorting to each access pattern.
```

Comment:
Note that the generated packed variables retained the split_var metacomment
as they remain eligable to be split further into smaller pieces accorting
to the access patterns.